css-images-4/Overview.bs

```diff
@@ -2420,7 +2420,8 @@ Interpolating ''stripes()'' {#interpolating-stripes}
 
 	1. Both the starting and ending image must have the same number of <<color-stripe>>s.
 
-	2. Neither image uses a combination of <<length>>, <<percentage>>, and <<flex>> stripes.
+	2. Each pair of interpolated thicknesses must be of the same type,
+		i.e. both must either be of type <<length>, <<percentage>>, or <<flex>>.
```

Comment:
Missing `>` in `<<length>`.

And the grammar uses `<length-percentage> | <flex>`, so it should probably be

```suggestion
		i.e. both must either be of type <<length-percentage>> or <<flex>>.
```

(which implies that you lengths and percentages can be mixed).

Reply:
Changed. And as noted, I'll open a new issue to discuss mixing _all_ values.